mod_regression/models.py

```diff
@@ -95,6 +95,8 @@ class RegressionTest(Base):
     expected_rc = Column(Integer)
     active = Column(Boolean(), default=True)
 
+    last_passed_on = Column(Integer, ForeignKey('test.id', onupdate="CASCADE", ondelete="SET NULL"))
```

Comment:
Any reason for the extra line between the other column definition and this one?

Reply:
Also, we were going to show the last commit that passed but we are going to test ID that last passed?

Reply:
> Also, we were going to show the last commit that passed but we are going to test ID that last passed?

Yes I thought test ID would be more relevant, and it would need just 1 extra click to reach the commit

Reply:
> Any reason for the extra line between the other column definition and this one?

Removed